### ISSUE TITLE
Action bar covering the editor

### ISSUE THREAD

**Issue by kraftbj** [MichaelArestad/Press-This#10]
![screen shot 2014-10-15 at 07 36 43](https://cloud.githubusercontent.com/assets/88897/4646516/fd11ec88-5473-11e4-91ba-e64969660666.png)

When trying to type notes Press This that exceed the height of the window, the cursor types behind the action div.

**Comment by stephdau** [comment]
Any CSS-based resolution to this?

**Comment by MichaelArestad** [comment]
No. The full editor does the same thing. For whatever reason, iOS lets you just keep going even if the text is off the screen.
